presto-orc/src/main/java/com/facebook/presto/orc/TupleDomainFilterUtils.java

```diff
@@ -338,4 +403,24 @@ public static TupleDomainFilter toBigintValues(long[] values, boolean nullAllowe
 
         return BigintValuesUsingBitmask.of(min, max, values, nullAllowed);
     }
+
+    public static TupleDomainFilter toBigintValuesExclusive(long[] values, boolean nullAllowed)
```

Comment:
this is a copy-paste from toBigintValues; I think it would be better to use a flag in BigintValuesUsingXxx classes and pass that flag to toBigintValues.